packages/edit-site/src/components/save-button/index.js

```diff
@@ -52,12 +54,47 @@ export default function SaveButton( {
 				previewingThemeName: previewingTheme?.name?.rendered,
 			};
 		}, [] );
+
 	const { setIsSaveViewOpened } = useDispatch( editSiteStore );
+	const { customizedSaveButtonAction, customizedSaveButtonLabel } = useSelect(
+		( select ) => {
+			const { getSettings } = unlock( select( editSiteStore ) );
+			return {
+				customizedSaveButtonAction:
+					getSettings().__experimentalSaveButtonAction,
```

Comment:
I found an example of the action stored in the settings is called `__experimentalFetchUrlData`. As a result, it would be better to call it `__experimentalGetSaveButtonAction`. WDYT?